src/applications/facility-locator/components/search-results-items/Covid19Result.jsx

```diff
@@ -31,12 +42,8 @@ const Covid19Result = ({ location, index }) => {
           markerText={location.markerText}
         />
         <span
-          onClick={() => {
-            recordResultClickEvents(location, index);
-          }}
-          onKeyPress={() => {
-            recordResultClickEvents(location, index);
-          }}
+          onClick={clickHandler}
+          onKeyDown={clickHandler}
```

Comment:
`onKeyPress` is deprecated. Moving to `onKeyDown` as it processes **slightly** faster than `onKeyUp`.